app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    page_title.empty? ? base_title : page_title + " | " + base_title
```

Comment:
phân biệt empty?, nil?, blank?

Reply:
* `nil?` check nếu một giá trị là `nil` hoặc không
* `empty?` trong trường hợp này kiểm tra một string có phải là `""` hay không, vì thế, nếu `page_title == " "` thì `page_title.empty? #=> false`
* `blank?` ngoài kiểm tra string rỗng, còn kiểm tra nếu string chỉ chứa toàn ký tự whitespace

Reply:
Vậy chỗ này em sửa lại thành `blank?` ạ
Fixed